src/providers/tpm_provider/utils.rs

```diff
@@ -90,6 +90,7 @@ pub fn to_response_status(error: Error) -> ResponseStatus {
 #[derive(Serialize, Deserialize)]
 pub struct PasswordContext {
     pub context: TpmsContext,
+    /// This value is confidential and needs to be zeroized by its new owner.
     pub auth_value: Vec<u8>,
```

Comment:
Agree, will try to change to it!

Reply:
> does it not work with serializing

I first tried to put the `auth_value` in a `Secret` but it did not work because there is no way to take the ownership out of a `Secret` and  that is needed to build an [`Auth`](https://docs.rs/tss-esapi/4.0.8-alpha.1/tss_esapi/structures/struct.Auth.html) (through a `TryFrom`).

Then I thought it would be best to directly use the `tss_esapi` `Auth` structure there directly. But that structure does not implement `Serialize`, `Deserialize` so it does not work.

I think I will leave how it is right now but we maybe should put the `Auth` value in the tss crate as a `Secret` and use it directly from there? It could be done as part of parallaxsecond/rust-tss-esapi#46

Reply:
That sounds good!